backend/apid/graphql/mutations.go

```diff
@@ -139,7 +139,7 @@ func (r *mutationsImpl) ResolveEvent(p schema.MutationResolveEventFieldResolverP
 		event.Check.Output = "Manually resolved manually with " + p.Args.Input.Source
```

Comment:
Sorry, I know this isn't related, but "manually resolved manually" is bothering me..

Reply:
🤦‍♂️ thanks. I'll fix that.